app/models/like.rb

```diff
@@ -1,2 +1,19 @@
+# == Schema Information
+#
+# Table name: likes
+#
+#  id         :bigint           not null, primary key
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  author_id  :bigint           not null
+#  post_id    :bigint           not null
+#
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, counter_cache: true
+
+  # updates likes counter
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
**OPTIONAL**

- [ ] It will be nice to use the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes_counter` method every time a new like has been saved to the database.

- [ ] Also make the `update_liks_counter` method private.